generator/internal/http_option_utils.cc

```diff
@@ -54,12 +54,12 @@ std::string FormatFieldAccessorCall(
   return absl::StrJoin(chunks, "().");
 }
 
-void RestPathVisitorHelper(std::string const& api_version,
+void RestPathVisitorHelper(absl::optional<std::string> const& api_version,
```

Comment:
Should we avoid using `const&` with `absl::optional`?

Reply:
You're right. We should take by value and `std::move()` the object.